Gemfile

```diff
@@ -39,6 +39,7 @@ gem "uglifier"
 gem "whenever", require: false
 
 group :test do
+  gem "capybara-select-2"
```

Comment:
The link in the commit message leads to a "There is nothing to compare" page 🤔  https://github.com/argerim/select2-rails/compare/v3.5.9.1...v3.5.10diff-ffea88f08f5750e44aab693b050ad1f4e03fc54ec95a48f814dd67395ec0df95R886

Reply:
It looks like GitHub removes the hash symbol from the URL when rendering the commit message in the UI.

The [full URL](https://github.com/argerim/select2-rails/compare/v3.5.9.1...v3.5.10#diff-ffea88f08f5750e44aab693b050ad1f4e03fc54ec95a48f814dd67395ec0df95R886) is there when the repo is pulled and the git log inspected.

Reply:
Ahhh, interesting! Sounds like a [ways to improve GitHub](https://twitter.com/search?q=Ways%20to%20improve%20%40github%20(from%3Achrisbashton)&src=typed_query&f=live) suggestion to me...! 🤔